### PR TITLE
added show wlan sum template

### PR DESCRIPTION
**Pull request by timjsmith24**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

cisco_wlc_ssh_show_wlan_sum.textfsm, .*, cisco_wlc_ssh, show wlan summary

##### SUMMARY

New template

```

```

### CODE REVIEW

File: templates/cisco_wlc_ssh_show_wlan_sum.textfsm [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Value SSID (.+?)
+Value STATUS (Enabled|Disabled)
+Value INTERFACE (\w+(.?\w+)+)
+
+
+Start
+  ^Number\s+of\s+WLANs
+  ^WLAN\s+ID\s+WLAN\s+Profile\s+Name
+  ^-------\s+
+  ^${WLANID}\s+${PROFILE}\s/\s${SSID}\s+${STATUS}\s+${INTERFACE}\s* -> Record
```
Comment (jmcgill298) [inline templates/cisco_wlc_ssh_show_wlan_sum.textfsm]:
```suggestion
  ^${WLANID}\s+${PROFILE}\s/\s${SSID}\s+${STATUS}\s+${INTERFACE}\s*$$ -> Record
```

File: templates/cisco_wlc_ssh_show_wlan_sum.textfsm [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Value WLANID (\d+)
+Value PROFILE (.+?)
+Value SSID (.+?)
+Value STATUS (Enabled|Disabled)
+Value INTERFACE (\w+(.?\w+)+)
+
+
+Start
+  ^Number\s+of\s+WLANs
+  ^WLAN\s+ID\s+WLAN\s+Profile\s+Name
```
Comment (jmcgill298) [inline templates/cisco_wlc_ssh_show_wlan_sum.textfsm]:
```suggestion
  ^WLAN\s+ID\s+WLAN\s+Profile\s+Name\s*$$
```

File: templates/cisco_wlc_ssh_show_wlan_sum.textfsm [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  ^-------\s+
+  ^${WLANID}\s+${PROFILE}\s/\s${SSID}\s+${STATUS}\s+${INTERFACE}\s* -> Record
+  ^\s*$$
+  ^. -> Error
```
Comment (jmcgill298) [inline templates/cisco_wlc_ssh_show_wlan_sum.textfsm]:
```suggestion
  ^. -> Error
  
```

**Comment by FragmentedPacket** [comment]
@timjsmith24 Just wanted to bump this so we can get it merged in sooner than later.

**Pull request closed without merging**